### PR TITLE
[ADD] web_widget_url_advanced

### PR DESCRIPTION
**Pull request by simahawk**
Upstream PR on master https://github.com/odoo/odoo/pull/28830

### CODE REVIEW

**Review by leemannd — APPROVED** [review]
Approving it.
Non-blocking comment:
- in order to revew IU changes, it could be great to add a screenshot about your code

**Comment by pedrobaeza** [comment]
Or add in demo data an example over res.partner with field name as source

**Review by guewen — APPROVED** [review]

**Comment by simahawk** [comment]
TBH you need very little imagination to have an idea of "you see the other field value instead of the link"... anyway I'll add a screenshot :stuck_out_tongue_winking_eye:

**Comment by pedrobaeza** [comment]
Indeed, but demo data will help you to test the feature.

**Comment by nikul-serpentcs** [comment]
@simahawk IMO Also check with Jslint

https://travis-ci.org/OCA/web/jobs/457826837#L1571-L1573

**Review by moylop260 — APPROVED** [review]
The jslint are optional.

👍 
Could you review why the README is not showing the images, please?

https://github.com/OCA/web/blob/6a31c51cc1b4b36447c96ade90747b8df39f8bbe/web_widget_url_advanced/README.rst

**Comment by simahawk** [comment]
@moylop260 don't know what do here... as usual, either it works w/ index.html or w/ README.rst or w/ the app description. Any silver bullet? I'd use absolute URLs but I don't know what's the best practice in this case :man_shrugging: 
TBH this feature is so simple that I'd remove them... :stuck_out_tongue_winking_eye:

File: web_widget_url_advanced/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <field name="website" widget="url" text_field="name" placeholder="e.g. www.odoo.com"/>
+
+This is going to change the rendering from
+
+.. figure:: /static/description/before.png
+
+
+to
+
+.. figure:: /static/description/after.png
```
Comment (sbidoul) [inline web_widget_url_advanced/readme/USAGE.rst]:
```suggestion
.. figure:: ../static/description/after.png
```

File: web_widget_url_advanced/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+if you want to display the website link with partner's name as anchor you can do:
+
+
+.. code-block:: xml
+
+    <field name="website" widget="url" text_field="name" placeholder="e.g. www.odoo.com"/>
+
+This is going to change the rendering from
+
+.. figure:: /static/description/before.png
```
Comment (sbidoul) [inline web_widget_url_advanced/readme/USAGE.rst]:
```suggestion
.. figure:: ../static/description/before.png
```

**Review by sbidoul — COMMENTED** [review]
The best way to use images in readme, thanks to https://github.com/OCA/maintainer-tools/pull/397

**Comment by ovnicraft** [comment]
@simahawk possible resolve @sbidoul comments ?

**Comment by simahawk** [comment]
fixed, rebased, squashed

**Review by nikul-serpentcs — APPROVED** [review]
If Jslint is optional then LGTM

[Travis Jslint](https://travis-ci.org/OCA/web/jobs/476867651#L1498-L1501)

**Pull request merged**